static/js/list_render.js

```diff
@@ -295,13 +295,8 @@ var list_render = (function () {
         // add built-in generic sort functions.
         prototype.add_generic_sort_function("alphabetic", function (prop) {
             return function (a, b) {
-                if (a[prop] > b[prop]) {
-                    return 1;
-                } else if (a[prop] === b[prop]) {
-                    return 0;
-                }
-
-                return -1;
+                // Sorts alphabetic lists without considering the case.
+                return a[prop].localeCompare(b[prop]);
```

Comment:
localeCompare is really slow; I'm reluctant to use it here.  I'm not sure whether `util.strcmp` is case-insensitive, though.

Reply:
I've reverted back to (almost) the same code as before, converting the strings to uppercase now before comparing them.